src/infrastructure/group-store/group-store.test.ts

```diff
@@ -52,6 +52,17 @@ describe("test groups stores", () => {
     const nameHash3 = keccak256(toUtf8Bytes(savedGroup3.name + "/" + "2"));
     const savedId3 = BigNumber.from(nameHash3).mod(UINT128_MAX).toHexString();
     expect(savedGroup3.id).toBe(savedId3);
+
+    const savedGroup4 = await groupStore.save(testGroups.group1_2);
```

Comment:
same here what is the purpose of the test?

Reply:
Same than this answer: https://github.com/sismo-core/sismo-hub/pull/2027/files#r1273486184